bulk/cli/util.py

```diff
@@ -25,3 +30,20 @@ def concat(
     if shuffle:
         df = df.sample(frac=1)
     df.drop_duplicates().to_csv(out, index=False)
+
+@app.command("resize")
+def resize(
+        path: pathlib.Path = typer.Argument(..., help="Path to .csv/.jsonl file", exists=True),
+        thumbnail_paths: pathlib.Path = typer.Argument(..., help="Path to folder to store thumbnails.", exists=False)):
+    """Resize the images into thumbnails."""
+
+    os.system(f"mkdir {thumbnail_paths}")
```

Comment:
I prefer to use `Pathlib` here, but that's also something I can do post-PR.

Reply:
Same with the comment on the prompt, by the way. That's also something I can pickup.